### PR TITLE
Coerce string values to numeric types

### PR DESCRIPTION
**Pull request by martint**

### CODE REVIEW

File: presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/decoders/BigintDecoder.java [hunk 1]
```diff
@@ -44,6 +44,9 @@ public void decode(SearchHit hit, Supplier<Object> getter, BlockBuilder output)
         else if (value instanceof Number) {
             BIGINT.writeLong(output, ((Number) value).longValue());
         }
+        else if (value instanceof String) {
+            BIGINT.writeLong(output, Long.parseLong((String) value));
```
Comment (dain) [inline presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/decoders/BigintDecoder.java:48]: Maybe catch the parse error and throw a better exception?

File: presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/decoders/IntegerDecoder.java [hunk 2]
```diff
@@ -45,6 +45,9 @@ public void decode(SearchHit hit, Supplier<Object> getter, BlockBuilder output)
         else if (value instanceof Number) {
             INTEGER.writeLong(output, toIntExact(((Number) value).longValue()));
         }
+        else if (value instanceof String) {
+            INTEGER.writeLong(output, Integer.parseInt((String) value));
```
Comment (dain) [inline presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/decoders/IntegerDecoder.java:49]: Maybe parse as long and then apply int exact?

**Review by dain — APPROVED** [review]
Consider improving error handling in these

**Comment by martint** [comment]
@dain, updated

**Review by dain — APPROVED** [review]

**Pull request merged**